docs/release-notes/release-notes-0.15.1.md

```diff
@@ -120,6 +120,8 @@ default](https://github.com/lightningnetwork/lnd/pull/6810) in most cases.
   `SendToRoute`](https://github.com/lightningnetwork/lnd/pull/6545). Set it to
   true so the payment won't be failed unless a terminal error has occurred,
   which is useful for constructing MPP.
+  
+* [Add list addresses RPC](https://github.com/lightningnetwork/lnd/pull/6596).
```

Comment:
Unfortunately we missed 0.15.1 as it's already in RC now. Can you update this to 0.16 instead please?

Reply:
Sure!